browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -571,4 +579,22 @@ TEST_F(BraveWalletServiceUnitTest, GetAndSetDefaultWallet) {
   EXPECT_EQ(GetDefaultWallet(), mojom::DefaultWallet::Ask);
 }
 
+TEST_F(BraveWalletServiceUnitTest, RecordWalletHistogram) {
+  service_->RecordWalletUsage(base::Time::Now());
```

Comment:
you could just create the histogram tester on stack in this test

Reply:
if we have any browser tests we could also check this metrics, at least on the service creation